scripts/Run-Razor-Locally.ps1

```diff
@@ -6,4 +6,4 @@ param()
 
 . $PSScriptRoot\Custom-Hive.ps1
 
-Test-Template "webapp" "webapp -au Individual" "Microsoft.DotNet.Web.ProjectTemplates.2.2.2.2.0-preview3-t000.nupkg" $false
+Test-Template "webapp" "webapp -au Individual" "Microsoft.DotNet.Web.ProjectTemplates.2.2.2.2.0-rtm-t000.nupkg" $false
```

Comment:
@ryanbrandenburg this version seems wrong.

Reply:
No, this is correct. The package is `Microsoft.DotNet.Web.ProjectTemplates.2.2`, the version is `2.2.0-rtm-t000`. If you look at the names of the nupkg's that templating dumps in artifacts/build you'll see that they look just like this.

Reply:
So, for 3.0 it should be `Microsoft.DotNet.Web.ProjectTemplates.3.0.3.0.0-alpha1-t000.nupkg`

Reply:
I see. Confusing 😄

Reply:
I agree, we talked about changing it but the specifics of the templating CLI conspire against us ☹️.